sentry/src/main.rs

```diff
@@ -98,6 +98,8 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
 
     let logger = logger();
     let redis = redis_connection().await?;
+    // setup migrations before setting up Postgres
+    setup_migrations().await;
```

Comment:
`bb8` doesn't  check if the credentials are correct, I haven't diged into this, but it usually times out on the first request, so it's good that we first setup the migrations which will check the credentials.